pkg/datapath/loader/loader_test.go

```diff
@@ -11,14 +11,15 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-// +build !privileged_tests
+//
+// +build privileged_tests
```

Comment:
Out of curiosity, why does it need to be privileged?

Reply:
Because it's creating devices and loading programs onto them. At the very least the `bpf()` syscall requires root privileges for what we're doing here.

Reply:
@joestringer I should have been more clear. The remove `//` was to be replaced with a new line. Just that the copyright is detached from the `build`

Reply:
I thought that's what I did, but I did it via GH's new change suggestion tool which did this.

Anyway, what difference does it make?

Reply:
`+build` might be considered part of the copyright header when modifying the copyright header

Reply:
I fixed this up in the new version.